packages/alfa-rules/src/sia-r46/rule.ts

```diff
@@ -44,7 +44,7 @@ export default Rule.Atomic.of<Page, Element>({
                 // The table model only works if the element is a th.
                 hasName("th"),
                 hasRole("rowheader", "columnheader"),
-                isPerceivable(device)
+                not(isIgnored(device))
```

Comment:
This should still use `isPerceivable()`.

Reply:
That was a conscious choice because the rule (https://act-rules.github.io/rules/d0f69e) is requiring both the `th` and its `table ancestor to be "included in the accessibility tree" and I guess they should be handled the same way.

Difference will be for something like
```html
<table>
  <tr>
    <th role="none"> <img src="foo.jpg" alt="Foo" /></th>
  <tr>
</table>
```

🤔 Looking at it now, I wonder if the ACT rule is correct in ignoring that…
I'll file an issue on ACT rules so at least we know what we should do.

Reply:
Checking with NVDA, the header is not announce in that case. So it is correct to skip this.

This is already Inapplicable due the role on the `th`. I am not sure it is possible to exclude an "element with a role of `row/columnheader`" from the accessibility tree without making its content accessible (I can see how to exclude it by means of `display: none` or `aria-hidden` but both these methods are inherited to descendants).

So I am not sure it really makes a difference, and the NVDA test make me believe that `not(isIgnored)` is correct.

Reply:
The header is required to be visible for it to be applicable, which is also checked by `isPerceivable()`.